tech/tech/interfaces/schemas/product_schema.py

```
@@ -0,0 +1,15 @@
+from pydantic import BaseModel, ConfigDict
+from typing import Literal
+
+class ProductSchema(BaseModel):
```

Ruff (0.8.2)

Expected 2 blank lines, found 1

Add missing blank line(s)

(E302)